X10D/src/Reflection/MemberInfoExtensions.cs

```diff
@@ -37,8 +37,8 @@ public static bool HasCustomAttribute<T>(this MemberInfo member)
             throw new ArgumentNullException(nameof(member));
         }
 #endif
-
-        return member.HasCustomAttribute(typeof(T));
+        
```

Comment:
Can you remove this whitespace on line 40? The Source Validator job will fail if it sees trailing whitespace like this.